src/esp/sim/Simulator.cpp

```diff
@@ -200,6 +196,11 @@ void Simulator::reconfigure(const SimulatorConfiguration& cfg) {
 #endif
     renderer_->acquireGlContext();
   }
+  // load IBL assets if appropriate and not loaded already
+  // TODO : So many things.  Needs to be config driven, for one.
```

Comment:
Hm, given you're replacing the "ground truth" test images here, I'm thinking it could make sense to load an actual HDR image here, not a LDR. Which again changes the intensity / balance a bit, but closer to the ideal I think.

As far as I remember, the JPEG was a temporary thing Yili did because at that point EXR loading wasn't done from my side yet. But now it is and -- as far as I'm aware -- is also built & enabled inside Habitat, and loading it here instead of the JPEG should hopefully "just work". The `*.exr` can be downloaded from here: https://polyhaven.com/a/lythwood_room

Reply:
Unfortunately, the exr file is 17 mb, too big without LFS.  Do we want to force LFS on folks to get anything IBL working?  

As far as HDR images are concerned, I already have a few that I've been playing with, I just wanted to keep the changes minimal with this PR.  There's going to be a follow-up (after the PBR shader work gets merged in, ideally) that will include a more rigorous redesign of the IBL subsystem. (i.e. config-driven for one)

Reply:
To be honest, the jpg seems to me to be working fine, at least for demonstration purposes.

Reply:
You can make it smaller with `magnum-imageconverter`. I don't think a 4K image is needed for our use case, the below shrinks the file to 5.7 MB. 

```sh
magnum-imageconverter lythwood_room_4k.exr -C StbResizeImageConverter -c size="1024 512" out.exr
```

It could be also converted from 32-bit floats to half-floats to drop ~half of the data but you'd have to write a Python script for that -- the EXR plugins don't have any convenience toggles for data conversion right now.

> I just wanted to keep the changes minimal with this PR

I only mentioned this because I saw the changes in test images, and the LDR JPEG is really not a good option so for me this felt like a good opportunity to switch.

Reply:
Cool, thanks!  I'll do this.

Reply:
For anyone else tracking this, https://github.com/mosra/magnum-plugins/commit/b8a1f670b21d2347515ccf707f5d42cb79efd810 and https://github.com/mosra/magnum-ci/actions/runs/5115547986 has a new option to import 32-bit EXRs as 16-bit, which is usable for significant size reduction.